File: app/src/main/java/hsm/yeng/join/ViewPagerAdapter.java
package hsm.yeng.join;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

/**
 * Created by arunima on 30/11/17.
 */

public class ViewPagerAdapter extends FragmentPagerAdapter{

    public ViewPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        Fragment fragment=null;
        if(position==0){
            fragment=new YengJoinFragment();
        }else if(position==1){
            fragment=new CAJoinFragment();
        }
        return fragment;
    }

    @Override
    public int getCount() {
        return 2;
    }

    @Override
    public CharSequence getPageTitle(int position) {
        switch (position){
            case 0:
                return "Yeng";
            case 1:
                return "Crazy Amigos";
        }
        return super.getPageTitle(position);
    }
}

File: app/src/main/java/in/yeng/user/joinwithus/helpers/ViewpagerAdapter.kt
package `in`.yeng.user.joinwithus.helpers

import `in`.yeng.user.joinwithus.children.CrazyAmigosFragment
import `in`.yeng.user.joinwithus.children.YengFragment
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter

class ViewpagerAdapter(val fragmentManager: FragmentManager) : FragmentStatePagerAdapter(fragmentManager) {

    companion object {
        val YENG = 0
        val CRAZY_AMIGOS = 1
    }

    override fun getItem(position: Int): Fragment =
            when (position) {
                YENG -> YengFragment()
                CRAZY_AMIGOS -> CrazyAmigosFragment()

                else -> YengFragment()
            }

    override fun getCount(): Int = 2

    override fun getPageTitle(position: Int): CharSequence? {
        return when(position)   {
            YENG -> "Yeng"
            CRAZY_AMIGOS -> "Crazy Amigos"

            else -> "Yeng"
        }
    }

}